deye_sensors.py

```diff
@@ -47,39 +47,48 @@
 pv1_current_sensor = SingleRegisterSensor("PV1 Current", 0x6e, 0.1, mqtt_topic_suffix='dc/pv1/current')
 pv1_power_sensor = ComputedPowerSensor("PV1 Power", pv1_voltage_sensor,
                                        pv1_current_sensor, mqtt_topic_suffix='dc/pv1/power')
+pv1_daily_sensor = SingleRegisterSensor("PV1 Daily", 0x41, 0.1, mqtt_topic_suffix='dc/pv1/daily')
```

Comment:
@Antagor This reports different type of information depending on the inverter type
1. String Inverter - Total Reactive Energy (Low word)
2. Hybrid Inverter - Month PV Energy
3. Micro Inverter - Power generation of component 1 on the day

Your case is 3.

I'm good with accepting this change, because it is not in conflict with already mapped register